gulpfile.js

```diff
@@ -1,21 +1,21 @@
 'use strict';
 
-var gulp = require('gulp');
-var sass = require('gulp-sass');
+const gulp = require('gulp');
+const sass = require('gulp-sass');
 
-var pathScss = './hkm/static/hkm/scss/**/*.scss';
-var pathCss = './hkm/static/hkm/css/';
-var pathStaticrootCss = './staticroot/hkm/css/';
+const pathScss = './hkm/static/hkm/scss/**/*.scss';
+const pathCss = './hkm/static/hkm/css/';
+const pathStaticrootCss = './staticroot/hkm/css/';
 
-gulp.task('sass', function () {
+gulp.task('sass', function() {
   return gulp.src(pathScss)
-    .pipe(sass().on('error', sass.logError))
```

Comment:
Here it seems like you are removing error logging. Can this make the error disappear and debugging harder? Or is the error just handled in a different way now?

Reply:
I forgot to add this back. Will commit it before merge.